lua/telescope/builtin/internal.lua

```diff
@@ -357,12 +390,22 @@ internal.help_tags = function(opts)
           table.insert(matches, match)
         end
 
-        if table.getn(matches) ~= 0 then
-          table.insert(tags, {
-            name = matches[1],
-            filename = all_help_files[matches[2]],
-            cmd = matches[3]
-          })
+        if #matches ~= 0 then
+          local found
+          for _, lang in ipairs(helplangs) do
+            if lang_tags[lang][matches[1]] then
+              table.insert(tags, lang_tags[lang][matches[1]])
+              found = true
+              break
+            end
+          end
+          if not found then
+            table.insert(tags, {
+              name = matches[1],
+              filename = all_help_files[matches[2]],
+              cmd = matches[3]
+            })
+          end
```

Comment:
What about when people have setup `vim.o.helplang` and still want the eng version at some point maybe because the translated version for something is not yet up to date? (Is that an issue) Should we do an opt so people can specify the preferred language

Reply:
Good catch. I'll add it.